bin/offline-cluster.sh

```diff
@@ -2,43 +2,58 @@
 
 set -eou pipefail
 
-SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-ANSIBLE_DIR="$( cd "$SCRIPT_DIR/../ansible" && pwd )"
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+ANSIBLE_DIR="$(cd "$SCRIPT_DIR/../ansible" && pwd)"
 
 set -x
 
 ls $ANSIBLE_DIR/inventory/offline
 
+if [ -f "$ANSIBLE_DIR/inventory/offline/hosts.ini" ]; then
+  INVENTORY_FILE="$ANSIBLE_DIR/inventory/offline/hosts.ini"
+elif [ -f "$ANSIBLE_DIR/inventory/offline/inventory.yml" ]; then
+  INVENTORY_FILE="$ANSIBLE_DIR/inventory/offline/inventory.yml"
+else
+  echo "No inventory file in ansible/inventory/offline/. Please supply an $ANSIBLE_DIR/inventory/offline/inventory.yml or $ANSIBLE_DIR/inventory/offline/hosts.ini"
+  exit -1
+fi
+
+if [ -f "$ANSIBLE_DIR/inventory/offline/hosts.ini" ] && [ -f "$ANSIBLE_DIR/inventory/offline/inventory.yml" ]; then
+  echo "Both hosts.ini and inventory.yml provided in ansible/inventory/offline! Pick only one."
+  exit -1
+fi
+
+echo "using ansible inventory: $INVENTORY_FILE"
+cat $INVENTORY_FILE
```

Comment:
Is this a debugging left-over? Or, should this be upstreamed?

Reply:
removing 👍